### ISSUE TITLE
Look at all pop-ups and see what love they need (add list here)

### ISSUE THREAD

**Issue by daver78** [coa0329/BhaangWeb#72]

**Comment by daver78** [comment]
@paagla123 Will you give me a list of the ones you find, and how to get them to occur?

**Comment by paagla123** [comment]
@daver78 here goes:
1) Success sending test email, which you already did.
2) PUBLISH SUCCESS when we successfully publish a campaign
3) Failure to publish when we have a problem in Studio

**Comment by paagla123** [comment]
@daver78 are you planning to look into this? Thx

**Comment by daver78** [comment]
@paagla123 Yes.  Is the list above everything we need to modify?  Or are there more.

**Comment by paagla123** [comment]
Provide message content to @coa0329 👍 
Create text/content for each of these for @coa0329 👍 

Success sending test email, which you already did.
PUBLISH SUCCESS when we successfully publish a campaign
Failure to publish when we have a problem in Studio
Update campaign - previously launched (Blog)

**Comment by coa0329** [comment]
@paagla123 

Please give me more information.

**Comment by daver78** [comment]
@paagla123 Tagging you in case you missed this.

**Comment by paagla123** [comment]
More pop-ups are getting added - I will deal with it altogether
